src/ID3Frames.js

```diff
@@ -545,3 +545,31 @@ module.exports.COMR = {
         return tag
     }
 }
+
+module.exports.GEOB = {
+    create: (data) => {
+        if (!(data instanceof Array)) {
+            data = [data]
+        }
+        const encoding = 3 // UTF-8
```

Comment:
We should insert here a validation:
- throw if content description is missing
- throw if object data buffer is missing
- throw if all content description do not have a unique value